contributions/api/controllers/contributions.py

```diff
@@ -0,0 +1,574 @@
+#  Copyright 2020 Board of Trustees of the University of Illinois.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import json
+import datetime
+import logging
+import copy
+
+from flask import jsonify, request, g
+from bson import ObjectId
+
+import controllers.configs as cfg
+import utils.jsonutils as jsonutils
+import utils.datasetutils as datasetutils
+import utils.rest_handlers as rs_handlers
+import utils.mongoutils as mongoutils
+
+from utils import query_params
+from models.contribution import Contribution
+from models.person import Person
+from models.organization import Organization
+from models.capabilities.capability import Capability
+from models.talents.talent import Talent
+from pymongo import MongoClient
+
+client_contribution = MongoClient(cfg.MONGO_CONTRIBUTION_URL, connect=False)
+db_contribution = client_contribution[cfg.CONTRIBUTION_DB_NAME]
+coll_contribution = db_contribution[cfg.CONTRIBUTION_COLL_NAME]
+
+def post():
+    is_new_install = True
```

Comment:
```suggestion
    is_new_install = True
    in_json = None
```
I think it's better to initialize in_json to None.

Reply:
added